src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs

```diff
@@ -159,6 +225,29 @@ private InlineDiffLine<CodeLine>[] CreateDiffOnlyLines(InlineDiffLine<CodeLine>[
                 }
             }
             return filteredLines.ToArray();
+
+            // The lines of code are already marked up with HTML which makes
+            // them a little difficult to process.  This strips out all of the
+            // markup and computes the size of the whitespace indent at the
+            // beginning of the line.
+            static (int indentSize, string code) GetIndentAndCode(string line)
+            {
+                if (string.IsNullOrEmpty(line)) { return (0, ""); }
+
+                // Parse the line as HTML
+                HtmlDocument html = new HtmlDocument();
```

Comment:
This feels very inefficient. We have an array of text-only-lines when calculating the diff.
``` C#
var diffLines = InlineDiff.Compute(
                    previousRevisionTextLines,
                    fileTextLines,
                    previousHtmlLines,
                    fileHtmlLines);
```


I think we just need to preserve them in the diff result.

Reply:
A simple way seems to be adding another property to InlineDiffLine and modifying https://github.com/Azure/azure-sdk-tools/blob/master/src/dotnet/APIView/APIView/DIff/InlineDiff.cs#L11 to include it.

Reply:
It's _super_ inefficient.  I rationalized it as it's only done for diff views (and it can handle Blobs fine), but I'll take a look at doing this higher in the stack when I get a chance.